guide/src/error_handling.md

```diff
@@ -65,13 +66,20 @@ when the container already exists.
 
 ## Handling the error
 
+Here's the code for updating a secret version:
+
+```rust,ignore
+{{#include ../samples/src/error_handling.rs:update-secret}}
+```
+
```

Comment:
We have been putting the full code blocks at the end. I think I would keep doing that.

Maybe at the bottom have an `update_secret` section along with the new `update_attempt`, `create_secret` sections.

Then here we would say something like:

```
We will write a function to update a secret, explaining each line of the
code. The full [`update_secret`](#update_secret) function is included below.

First make an attempt to create a new secret version:

... etc.
```

Reply:
Good feedback. Done. Thanks!